### ISSUE TITLE
vqmod for new version (opencart 3) ?

### ISSUE THREAD

**Issue by nikto1** [vqmod/vqmod#108]
As OC is also using twig now, it should be also for the (twig) engine. Many thanks

**Comment by qphoria** [comment]
twig is just text to vQmod. The current version of vQmod works fine on opencart 3.0 so far in my testing. You'll just need to be sure to update your vqmod xml scripts to support 3.0

**Comment by qphoria** [comment]
Ok I see there are some changes needed for the new Twig system as it has its own class structure. We have a preliminary fix and will release in the next couple of days.

**Comment by sculptex** [comment]
That's great, any chance you can post code fixes to get it to work with twig?
Of course delay release until you are happy, i'm happy to test!

**Comment by qphoria** [comment]
I've attached a working version of the vqmod_opencart.xml file
GitHub doesn't allow xml files for some reason and also seems to be blocking zip attachments even though they claim to be supported so I had to rename it to .txt
Rename it back to .xml and upload to your vqmod/xml folder
overwrite the old one. That should make it work with 3.0.0
Also backwards compatible with older versions of the cart.

[vqmod_opencart.txt](https://github.com/vqmod/vqmod/files/1092303/vqmod_opencart.txt)

**Comment by qphoria** [comment]
I've attached a working version of the vqmod_opencart.xml file
GitHub doesn't allow xml files for some reason and also seems to be blocking zip attachments even though they claim to be supported so I had to rename it to .txt.
Rename it back to .xml and upload to your vqmod/xml folder and
overwrite the old one. That should make it work with 3.0.0
Also backwards compatible with older versions of the cart.

[vqmod_opencart.txt](https://github.com/vqmod/vqmod/files/1092415/vqmod_opencart.txt)
